gto/tag.py

```diff
@@ -183,16 +183,26 @@ def find(
     return tags
 
 
-def create_tag(repo, name, ref, message):
+def create_tag(
+    repo: git.Repo,
+    name: str,
+    ref: str,
+    message: str,
+    GIT_COMMITTER_NAME: str = None,
+    GIT_COMMITTER_EMAIL: str = None,
```

Comment:
This still looks weird to me eye, though I haven't see all the code. What is the corresponding read function? This one is expected to receive the things that one is returning, i.e.
- if this is gto level abstraction it should be `author/auhtor_email`
- if this is git level then `tagger...` will be appropriate. 

`GIT_COMMITTER_NAME` is implementation detail of gitpython, so if this function is a wrapper to abstract that away it should not leak it like this. Imagine you were to replace gitpython with something else then this function will fail to contain the changes.